lls/main.c

```diff
@@ -834,6 +834,11 @@ run_lls(struct net_config *net_conf, struct lls_config *lls_conf)
 		goto out;
 	}
 
+	net_conf->gatekeeper_max_pkt_burst =
+		RTE_MAX(net_conf->gatekeeper_max_pkt_burst,
+			RTE_MAX(lls_conf->front_max_pkt_burst,
+				lls_conf->back_max_pkt_burst));
```

Comment:
We have to check that the back interface is enabled before accounting it in `gatekeeper_max_pkt_burst`.